sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala

```diff
@@ -87,6 +87,13 @@ class DiscoveryImpl(
       in.protocolMajorVersion,
       in.protocolMinorVersion)
 
+    if (in.devMode && BuildInfo.runtimeVersion.compareTo(in.proxyVersion) > 0) {
+      log.warn(
+        "Your service is using an outdated runtime image (version: {}). It's recommended to update your image to '{}' in your docker-compose.yml",
+        in.proxyVersion,
+        s"${BuildInfo.runtimeImage}:${BuildInfo.runtimeVersion}")
+    }
```

Comment:
This is the main change, the other changes in this file are more about renaming 'proxy' to 'runtime'.